tests/functional/test_report_only_no_errors.py

```diff
@@ -0,0 +1,319 @@
+# -*- coding: utf-8 -*-
+"""Report only submission
+
+Partners or redhat associates can publish their chart by submitting
+error-free report that was generated by chart-verifier.
+"""
+import os
+from dataclasses import dataclass
+from string import Template
+import tempfile
+import json
+import base64
+import git
+import pathlib
+from retrying import retry
+import logging
+import yaml
+from .utils import get_name_and_version_from_report
+
+import requests
+import pytest
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+    parsers,
+)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+@retry(stop_max_delay=30_000)
+def get_run_id(headers, secrets):
+    r = requests.post(
+        f'https://api.github.com/repos/{secrets.test_repo}/pulls/{secrets.pr_number}', headers=headers, json={})
+    pr = json.loads(r.text)
+
+    r = requests.get(
+        f'https://api.github.com/repos/{secrets.test_repo}/actions/runs', headers=headers)
```

Comment:
yes it will accumulate overtime.. maybe we should add a cleanup mechanism to cleanup testing workflow runs, and PR's.
Currently the commit is clean with `git push -f` during the latest test run, and the fork branch and testing gh-pages are cleaned after each test run.

Reply:
I there a specific resource to get runs per PR instead of all runs?

Reply:
Make the max time quite long - tests that can time out occasionally are a pain.

Reply:
From GitHub API doc it seems like they only support getting workflow run with run_id: https://docs.github.com/en/rest/reference/actions#get-a-workflow-run